src/assessment.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = True;
 //
 
-// Which scope(s) may access the variable `laswerQuantity`? Choose one or many
+// Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [deathStar,];
```

Comment:
The `deathStar` scope does have access to `laserQuantity`. However, because the `fire` function is defined within the `deathstar` scope, everything in _its_ scope will also have access to `laserQuantity`.